sdk/storage/Azure.Storage.Common/api/Azure.Storage.Common.netstandard2.0.cs

```diff
@@ -12,6 +12,10 @@ public enum ClientSideEncryptionVersion
     {
         V1_0 = 1,
     }
+    public static partial class StorageServerTimeout
```

Comment:
Should we make this more general so we can put other things here in the future?  What about something like `StorageExtensions.CreateServiceTimeoutScope()` that gives us a place to hang extension methods too?